packages/sdk/utils/src/account.ts

```diff
@@ -54,24 +55,74 @@ export async function generateMnemonic(
   return bip39ToUse.generateMnemonic(strength, undefined, getWordList(language))
 }
 
-export function validateMnemonic(
-  mnemonic: string,
-  defaultLanguage?: MnemonicLanguages,
-  bip39ToUse: Bip39 = bip39Wrapper
-) {
-  const mnemonicWords = mnemonic.trim().split(' ')
-  const languages = defaultLanguage
-    ? [defaultLanguage]
-    : getAllLanguages().filter((lang) => lang !== defaultLanguage)
+export function validateMnemonic(mnemonic: string, bip39ToUse: Bip39 = bip39Wrapper) {
+  const languages = getAllLanguages()
   for (const language of languages) {
-    const wordList = getWordList(language)
-    if (mnemonicWords.every((word) => wordList.includes(word))) {
-      return bip39ToUse.validateMnemonic(mnemonic, getWordList(language))
+    if (bip39ToUse.validateMnemonic(mnemonic, getWordList(language))) {
+      return true
     }
   }
+
   return false
 }
 
+export function formatNonAccentedCharacters(mnemonic: string) {
+  const languages = getAllLanguages()
+  const normMnemonicArr = normalizeAccents(mnemonic)
+    .trim()
+    .split(' ')
```

Comment:
One, tiny detail here, is that we should lower case the mnemonic just in case one of the words has an upper case char, otherwise we won't be able to find the word in the normalized-map
This should work:
```suggestion
  const normMnemonicArr = normalizeAccents(mnemonic)
    .toLowerCase()
    .trim()
    .split(' ')
```

In the case of the dictionary, we already know that every word is "lowercased", so we don't really need to add it there